gridsome.config.js

```diff
@@ -57,6 +57,22 @@ module.exports = {
           },
         ],
       },
+      {
+        name: "troubleshooting",
+        sections: [
+          {
+            title: "Troubleshooting your solutions",
```

Comment:
There is a repetition between the name of the sidebar and the title of the index page. Not sure how to reanme the sidebar (or the titale of the index...). Any suggestions?